### PR TITLE
Manage documents

### PR DESCRIPTION
**Pull request by james**
Redesign the manage documents index page

### CODE REVIEW

**Comment by james** [comment]
[Manage documents](https://trello.com/c/5xO0Nejl/40-manage-documents)

File: app/views/drawings/index.html.erb [hunk 1]
```diff
@@ -30,36 +31,41 @@
...
-      <% drawings.each.with_index do |drawing, index| %>
-        <div class="govuk-grid-column-one-half">
-          <div class="drawing-card<%= " right" if index % 2 == 1 %>">
-            <p class="govuk-body">
-              <strong><%= drawing.name %></strong><br/>
-              <%= drawing.created_at.strftime("%e %B %Y") %><br/>
-              <%= link_to "View in new window",
-              rails_blob_path(drawing.plan), target: :_new %><br/>
-            </p>
+<ul class="app-task-list current-drawings">
```
Comment (Rhian) [inline app/views/drawings/index.html.erb:34]: Unless I'm missing something, we've removed current-drawings from the custom CSS, so should we remove it from the <ul>?

**Review by Rhian — APPROVED** [review]
Looks good to me.

File: app/views/drawings/index.html.erb (hunk 1 above)
Comment (james) [reply app/views/drawings/index.html.erb:34]: It's being used by the tests, which isn't a great reason, but leaving semantic(ish) classes in the HTML doesn't upset me too much

**Pull request merged**